src/credentials/loginManager.ts

```diff
@@ -28,9 +28,10 @@ export class LoginManager {
      *
      * @param passive  If true, this was _not_ a user-initiated action.
      * @param provider  Credentials provider id
+     * @returns True if the toolkit could be logged in with the providerId
```

Comment:
Is it clearer with active voice?
```suggestion
     * @returns True if the toolkit could connect with the providerId
```